rpa-manager/modules/ROOT/pages/_partials/form-scheduling.adoc

```diff
@@ -21,33 +21,33 @@ Then, under *Repeats*, you can choose whether the *From-To* time interval is to
 +
 The duration of the contained time interval and the start time are taken into account. The duration of the time interval is defined as the difference between the *From* and *To* times. The *From* time is the start time of the first execution. The start time of repetitions depends on the other set values.
 +
-If you wish to execute the repeats only with reference to the time of day, you should ensure that the start and end date are identical. If the interval specified  covers multiple days, a new multi-day interval begins with every repetition. This can lead to a situation whereby the scheduled time interval lasts longer than expected because the full number of days contained in the *From-To* interval is executed once again during the last repetition.
+If you want to execute the repeats only with reference to the time of day, you should ensure that the start and end date are identical. If the interval specified  covers multiple days, a new multi-day interval begins with every repetition. This can lead to a situation whereby the scheduled time interval lasts longer than expected because the full number of days contained in the *From-To* interval is executed once again during the last repetition.
 +
 With schedules that are not all day, the actions in each repetition are executed for the duration of the interval specified in *From* and *To*.
 
 *** *Repeat minutely*:
 +
-If you select *Repeat minutely*, the planned action is executed every minute during the interval specified above.
+If you select *Repeat minutely*, the planned action is executed every minute during the interval specified via *To* and *From*.
```

Comment:
This is a preference - during the interval specified in the *From* and *To* fields.